src/System.Private.CoreLib/shared/System/Runtime/Intrinsics/Vector128.cs

```diff
@@ -2,43 +2,406 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using System.Diagnostics;
-using System.Runtime.CompilerServices;
-using System.Runtime.InteropServices;
 using Internal.Runtime.CompilerServices;
 
 namespace System.Runtime.Intrinsics
 {
-    [Intrinsic]
-    [DebuggerDisplay("{DisplayString,nq}")]
-    [DebuggerTypeProxy(typeof(Vector128DebugView<>))]
-    [StructLayout(LayoutKind.Sequential, Size = 16)]
-    public readonly struct Vector128<T> where T : struct
+    public abstract class Vector128
```

Comment:
Why `abstract class Vector128` and not `static class Vector128`?

Reply:
Just oversight on my end, will fix.